src/main/g8/src/main/scala/lib/$if(add_sql_orm.truthy)$sql-orm$endif$/domain/Example.scala

```diff
@@ -0,0 +1,3 @@
+package lib.sql_orm.domain
+
+final case class Example(id: String, value: Int)
```

Comment:
Maybe rename it to ExampleEntity or something more explicit about the fact it is an entity ;)